ios/browser/api/brave_shields/adblock_engine.mm

```diff
@@ -135,9 +135,10 @@ + (DomainResolverCallback)defaultDomainResolver {
   return brave_shields::AdBlockServiceDomainResolver;
 }
 
-+ (NSString*)contentBlockerRulesFromFilterSet:(NSString*)filterSet {
++ (NSString*)contentBlockerRulesFromFilterSet:(NSString*)filterSet
+                                    truncated:(bool*)truncated {
```

Comment:
@kylehickinson There is no need to use the objc BOOL here is there?

Reply:
`BOOL` actually just typedef's to `bool` for iOS so its fine to use `bool`

Reply:
We can give this a similar treatment to the blocking API though, and return an object instead of using inout pointers

Reply:
> return an object instead of using inout pointers

I'll leave that to you guys to either add onto this PR or in a followup then. I'm probably not the best to be making larger Objective C changes.

Reply:
Sure I can take care of it, but I'd like to do it as a separate PR if @kylehickinson has no objections